src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -335,6 +336,22 @@ private IEnumerable<TypeInferenceInfo> InferTypeInAnonymousObjectCreation(Anonym
                 return SpecializedCollections.EmptyEnumerable<TypeInferenceInfo>();
             }
 
+            private IEnumerable<TypeInferenceInfo> InferTypeInTupleExpression(
+                TupleExpressionSyntax tupleExpression, SyntaxToken previousToken)
+            {
+                if (previousToken == tupleExpression.OpenParenToken)
+                    return InferTypeInTupleExpression(tupleExpression, tupleExpression.Arguments[0]);
+
+                if (previousToken.Kind() == SyntaxKind.CommaToken)
+                {
+                    var argsAndCommas = tupleExpression.Arguments.GetWithSeparators();
+                    var commaIndex = argsAndCommas.IndexOf(previousToken);
+                    return InferTypeInTupleExpression(tupleExpression, (ArgumentSyntax)argsAndCommas[commaIndex + 1]);
```

Comment:
❔ Will this break for invalid syntax where the last argument is omitted?

Reply:
Will add test.  :)

Reply:
test added.